client/my-sites/customer-home/cards/tasks/growth-summit/index.jsx

```diff
@@ -11,10 +11,10 @@ const GrowthSummit = () => {
 		<Task
 			title={ translate( 'The WordPress.com Growth Summit' ) }
 			description={ translate(
-				'Learn how to build and grow your site, from start to scale. Come join us for this exclusive two-day virtual event, August 11-13.'
+				'Join us August 17th (Americas + EMEA) & August 18th (APAC) to get expert advice on how to grow your audience and earn money.'
 			) }
-			actionText={ translate( 'Register today for 20% off!' ) }
-			actionUrl="http://www.wordpress.com/growth-summit/"
+			actionText={ translate( 'Register today for $25 USD!' ) }
+			actionUrl="https://wordpress.com/growth-summit/?utm_source=my_home&utm_medium=organic_social_m[…]s=a8c-analytics.on,ad-tracking,google-analytics&ga_optimize=on"
```

Comment:
Is this link unintentionally truncated? Namely at `social_m[…]s=a8c`.

Reply:
Oops, good catch! Thank you ❤️